sw/host/ot_certs/src/asn1/x509.rs

```diff
@@ -230,6 +237,48 @@ impl X509 {
         })
     }
 
+    pub fn push_subject_alt_name_ext<B: Builder>(
+        builder: &mut B,
+        subject_alt_name: &IndexMap<AttributeType, Value<String>>,
+    ) -> Result<()> {
+        if subject_alt_name.is_empty() {
+            // Subject ALternative Name is an optional extension, it's ok not to be present.
+            return Ok(());
+        }
+        let required_attributes = [
```

Comment:
I would not add a list of required attributes, subject alt names can be used for everything and not just TPM. I think it's the responsability of the tpm template to set it up correct.

Reply:
ok, after all it is easy to check the produced certificates to see that the included fields are what's expected, dropped the checks.